bambu_spoolman/spoolman.py

```diff
@@ -105,12 +146,345 @@ def set_tray_uuid(self, spool_id, tray_uuid):
         # Set the new tray uuid
         extra[extra_field] = f"\"{tray_uuid}\""
         # Update the spool
-        response = requests.patch(
-            self._make_api_route(f"spool/{spool_id}"),
-            json={"extra": extra},
-            verify=self.verify,
-        )
-        return True if response.status_code == 200 else False
+        try:
+            response = requests.patch(
+                self._make_api_route(f"spool/{spool_id}"),
+                json={"extra": extra},
+                verify=self.verify,
+            )
+            response.raise_for_status()
+            return True
+        except requests.exceptions.HTTPError:
+            return False
+
+    def set_active_tray(self, spool_id, ams_num=None, tray_num=None):
+        """
+        Sets the AMS and tray fields for a spool
+
+        Args:
+            spool_id: The spool ID to update
+            ams_num: The AMS number (1-indexed), or None to clear
+            tray_num: The tray number (1-indexed), or None to clear
+
+        Uses environment variables:
+            SPOOLMAN_AMS_FIELD_NAME: Field name for AMS number (e.g., "ams_num")
```

Comment:
Can you update the Readme and `.env.example` to include these new environment variables with example usage. For the example env file, just comment them out like some of the other variables that already exist.